Tests/ResponseTests.swift

```diff
@@ -70,6 +70,8 @@ class ResponseTestCase: BaseTestCase {
         XCTAssertNil(response?.response)
         XCTAssertNil(response?.data)
         XCTAssertNotNil(response?.error)
+        XCTAssertEqual(response?.error?.isSessionTaskError, true)
+        XCTAssertEqual((response?.error?.underlyingError as? URLError)?.code, .cannotFindHost)
```

Comment:
Weird mismatch now between `response.error` and `result.failure`. I don't think I really like the `value` -> `success` and `error` to `failure` change. Those are the cases, not the values inside the cases. Seems like an odd choice to use the same name as a convenience property. Seems more like a Bool to me than a value.

Reply:
Covered in Slack discussion.